base/sparse/higherorderfns.jl

```diff
@@ -861,8 +862,10 @@ broadcast_indices(::Type{AbstractSparseArray}, A) = indices(A)
 # broadcast container type promotion for combinations of sparse arrays and other types
 _containertype{T<:SparseVecOrMat}(::Type{T}) = AbstractSparseArray
 # combinations of sparse arrays with broadcast scalars should yield sparse arrays
-promote_containertype(::Type{Any}, ::Type{AbstractSparseArray}) = AbstractSparseArray
-promote_containertype(::Type{AbstractSparseArray}, ::Type{Any}) = AbstractSparseArray
+promote_containertype(::Type{AbstractSparseArray}, ::ScalarType) = AbstractSparseArray
```

Comment:
I do not know whether sparse `map[!]/broadcast[!]` handles `Nullable`s correctly. (Hence the absence of `Nullable`s in the relevant code so far.) Have you confirmed that sparse `map[!]/broadcast[!]` handles `Nullable`s correctly? Thanks!